.github/workflows/dependencies-action.yml

```diff
@@ -1,3 +1,5 @@
+name: Dependency check
```

Comment:
With the job below named "Check Dependencies", wouldn't this show up as "Dependency check: Check Dependencies"?

How about we call the workflow "PR dependencies" and the job just "Check" ?